roles/cephadm/tasks/main.yml

```diff
@@ -4,23 +4,23 @@
 ---
 #- include_vars: "{{ seapath_distro }}.yml"
 
-#- name: Define CEPH_RELEASE variable
-#  set_fact:
-#    CEPH_RELEASE: "19.2.1"
-#
-#- name: Download cephadm
-#  get_url:
-#    url: "https://download.ceph.com/rpm-{{ CEPH_RELEASE }}/el9/noarch/cephadm"
-#    dest: "/usr/local/bin/cephadm"
-#    mode: '0755'
-#
-#- name: Add Ceph repository
-#  command: /usr/local/bin/cephadm add-repo --release squid
-#  changed_when: true
-#
-#- name: Install ceph-common package
-#  command: /usr/local/bin/cephadm install ceph-common ceph-volume
-#  changed_when: true
+- name: Define ceph_release variable
+  set_fact:
+    ceph_release: "19.2.1"
+
+- name: Download cephadm
+  get_url:
+    url: "https://download.ceph.com/rpm-{{ ceph_release }}/el9/noarch/cephadm"
```

Comment:
So, for the moment, we need an Internet connection on SEAPATH machines?

Reply:
not exactly, this is needed for the debian CI, which is installed for ceph-ansible.
If we consider that the user has the appropriate base installation, containing the ceph packages for a ceph-ansible deployment, or the cephadm package for the cephadm deployment, then we don't need this (and we also don't need the other task in the debian prerequisites playbook to remove the distro ceph packages).
I think it's a good assumption so I will remove those tasks, and mention something about that in the commit/PR.